Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Writers/CloudFormationJsonWriter.cs

```diff
@@ -12,47 +14,52 @@ public class CloudFormationJsonWriter : IAnnotationReportWriter
     {
         private readonly IFileManager _fileManager;
         private readonly IJsonWriter _jsonWriter;
+        private readonly IDiagnosticReporter _diagnosticReporter;
 
-        public CloudFormationJsonWriter(IFileManager fileManager, IJsonWriter jsonWriter)
+        public CloudFormationJsonWriter(IFileManager fileManager, IJsonWriter jsonWriter, IDiagnosticReporter diagnosticReporter)
```

Comment:
Does the `DiagnosticReporter` manipulate the `serverless.template` file? If not, then this should not be a part of `CloudFormationJsonWriter`.

Reply:
it doesn't manipulate but makes sure that serverless.template is in output.

Reply:
What do you mean by "_is in output_" ?

Reply:
https://github.com/aws/aws-lambda-dotnet/runs/4232177767?check_suite_focus=true#step:5:12067